ci-operator/config/stolostron/governance-policy-framework/stolostron-governance-policy-framework-main.yaml

```diff
@@ -73,6 +73,7 @@ tests:
           memory: 200Mi
     workflow: ocm-e2e-kind
 - as: test-e2e-grc-framework
+  run_if_changed: ^test/integration/.*
```

Comment:
Would this affect the periodic test runs also or just the PRs?

Reply:
Just the PRs. The periodic test runs in Travis.